### ISSUE TITLE
Path Parsing Error

### ISSUE THREAD

**Issue by william-silversmith** [seung-lab/cloud-files#56]
```python
from cloudfiles import CloudFiles
cf_local = CloudFiles("file:///...")
cf_cloudian = CloudFiles("matrix://something/test_gtbot/somethning2/c6005b4e897c0ab6")                                   
cf_local.transfer_to(cf_cloudian)
```
The script above will not copy the files to matrix://something/test_gtbot/something2/c6005b4e897c0ab6, instead everything is in matrix://something/test_gtbot/2/c6005b4e897c0ab6 , testing it with gs:// produces the same results.

**Comment by william-silversmith** [comment]
Thanks to @ranlu for reporting.

**Comment by william-silversmith** [comment]
Resolved in 33a7841ba5b03640527c70f864aff7d43fe00292
